futures-test/src/assert.rs

```diff
@@ -2,16 +2,38 @@ use futures_core::stream::Stream;
 use std::marker::Unpin;
 
 #[doc(hidden)]
-pub fn assert_is_unpin_stream<S: Stream + Unpin>(_: &S) {}
+pub fn assert_is_unpin_stream<S: Stream + Unpin>(_: &mut S) {}
 
 /// Assert that the next poll to the provided stream will return
 /// [`Poll::Pending`][futures_core::task::Poll::Pending].
+///
+/// # Examples
+///
+/// ```
+/// #![feature(async_await, futures_api, pin)]
+///
+/// #[macro_use]
+/// extern crate futures_test;
+///
+/// #[macro_use]
+/// extern crate futures;
```

Comment:
I think this isn't required. We rely on the Rust 2018 module system and there's also no macro from the futures crate in use

Reply:
`pin_mut!` is coming from `futures` still, this can change to `pin_utils` or whatever once that's here.

Reply:
ah right ^^'